src/scenes/world-map.js

```diff
@@ -201,6 +202,26 @@ export class WorldMapScene extends Phaser.Scene {
         this.events.on("shutdown", () => {
             this.scale.resize(SCREEN.WIDTH, SCREEN.HEIGHT);
         });
+
+        container.on("wheel", function (pointer, deltaX, deltaY, deltaZ) {
```

Comment:
Overall this is working but there are some weird edge cases at the edge of the map. I think it's good enough right now but worth keeping in mind

Reply:
Agreed. I struggled with some weird behavior for awhile and not sure what is the best solution for what we have, but this works good enough for me